src/app/page.tsx

```diff
@@ -1,23 +1,22 @@
 import { ProductGrid } from '@/components/ProductGrid';
 import { Filters } from '@/components/Filters';
+import Header from '@/components/header/Header';
+import HeroSection from '@/components/heroSection/HeroSection';
 
 export default function Home() {
   return (
-    <main className='flex min-h-screen flex-col items-center justify-between gap-6 p-6 md:gap-12 md:p-12'>
-      <header className='flex flex-col gap-4'>
-        <h1 className='text-3xl font-bold md:text-5xl'>
-          Mobile Phones & Accessories
-        </h1>
-        <p className='text-base md:text-2xl'>
-          Discover the latest mobile phones and accessories to enhance your
-          digital lifestyle. From sleek designs to powerful features, our
-          selection offers something for everyone.
-        </p>
-      </header>
-      <div className='w-full md:grid md:grid-cols-[1fr_4fr]'>
-        <Filters />
-        <ProductGrid />
-      </div>
-    </main>
+    <>
+      <Header />
+      <main className='p-main-content md:p-main-content-md lg:p-main-content-lg flex min-h-screen w-full flex-col items-center gap-y-10 lg:gap-y-12'>
+        <HeroSection
```

Comment:
🟡 This is more like a banner... hero section usually also has some heading and a CTA button. But how we end up naming this component is not that important.